extensions/notebook/package.json

```diff
@@ -465,13 +473,17 @@
           "command": "notebook.command.closeNotebook",
           "when": "view == bookTreeView && viewItem == savedNotebook"
         },
+        {
+          "command": "notebook.command.removeNotebook",
+          "when": "view == bookTreeView && viewItem == savedBookNotebook"
+        },
         {
           "command": "notebook.command.moveTo",
-          "when": "view == bookTreeView && viewItem == savedNotebook || view == bookTreeView && viewItem == section"
+          "when": "view == bookTreeView && viewItem == savedNotebook || view == bookTreeView && viewItem == savedBookNotebook || view == bookTreeView && viewItem == section"
         },
         {
           "command": "notebook.command.pinNotebook",
-          "when": "view == bookTreeView && viewItem  == savedNotebook",
+          "when": "view == bookTreeView && viewItem  == savedNotebook || view == bookTreeView && viewItem == savedBookNotebook",
```

Comment:
Similar to above `"view == bookTreeView && (viewItem == savedNotebook || viewItem == savedBookNotebook)"`

Reply:
I tried that but the `when` clause doesn't seem to support parentheses for grouping conditions 😞. If I use it then it only takes into account the last OR.
https://code.visualstudio.com/api/references/when-clause-contexts#conditional-operators